### DISCUSSION TITLE
Flows not displaying on the webpage

### DISCUSSION THREAD

**Discussion by agust-d23** [akvorado/akvorado#1327] [Q&A]
Hi @vincentbernat in my inlet file below I am mapping the public IP of my exporter device to its private IP, Would that effect the displaying of flows on the webpage? (i am not getting any flows at all at the moment). The reason I have mapped the public IP to it's private IP is because in my environment **the routers are unable to send netflow over management interface and the routers are accessible via SNMP only over management interface**. Appreciate your assistance on this :)

inlet file -

kafka:
  compression-codec: zstd

metadata:
  workers: 10
  providers:
    - type: snmp
      security-parameters:
        user-name: akvorado
        authentication-protocol: SHA
        authentication-passphrase: Password1
        privacy-protocol: AES
        privacy-passphrase: Password1
      agents:
        "publicIP of router": "privateIP of router"
        "publicIP of router": "privateIP of router"

flow:
  inputs:
    - type: udp
      decoder: netflow
      listen: ":2055"
      workers: 6
      receive-buffer: 10485760
    - type: udp
      decoder: sflow
      listen: ":6343"
      workers: 6
      receive-buffer: 10485760

core:
  workers: 6
  exporter-classifiers:
    # This is an example. This should be customized depending on how
    # your exporters are named.
    - ClassifySiteRegex(Exporter.Name, "^([^-]+)-", "$1")
    - ClassifyRegion("europe")
    - ClassifyTenant("acme")
    - ClassifyRole("edge")
  interface-classifiers:
    # This is an example. This must be customized depending on the
    # descriptions of your interfaces. In the following, we assume
    # external interfaces are named "Transit: Cogent" Or "IX:
    # FranceIX".
    - |
      ClassifyConnectivityRegex(Interface.Description, "^(?i)(transit|pni|ppni|ix):? ", "$1") &&
      ClassifyProviderRegex(Interface.Description, "^\\S+?\\s(\\S+)", "$1") &&
      ClassifyExternal()
    - ClassifyInternal()

please let me know if you need any clarification from my end, i'd be happy to help

**Comment by vincentbernat** [comment]
No, this should be OK. Have a look at the troubleshooting page. Notably, you didn't customize the classification, so it may not work.

**Reply by joelmoxey** [reply]
I am also doing something like this, but I see the SNMP query is not changed:

```
# docker compose exec akvorado-orchestrator akvorado orchestrator --check --dump /etc/akvorado/akvorado.yaml
<snip>
      metadata:
        cacheduration: 30m0s
        cacherefresh: 1h0m0s
        cachecheckinterval: 2m0s
        cachepersistfile: ""
        providers:
            - agents:
                212.183.148.10: 192.168.194.2
              communities:
                ::/0:
                    - snmpcommunity
<snip>

13:56:04.613288 vethf6db0f7 P   IP 172.19.0.7.50537 > 212.183.148.10.snmp:  GetRequest(128)  system.sysName.0 interfaces.ifTable.ifEntry.ifDescr.123 31.1.1.1.18.123 31.1.1.1.15.123 interfaces.ifTable.ifEntry.ifDescr.119 31.1.1.1.18.119 31.1.1.1.15.119
13:56:04.613288 br-1c94d99daebe In  IP 172.19.0.7.50537 > 212.183.148.10.snmp:  GetRequest(128)  system.sysName.0 interfaces.ifTable.ifEntry.ifDescr.123 31.1.1.1.18.123 31.1.1.1.15.123 interfaces.ifTable.ifEntry.ifDescr.119 31.1.1.1.18.119 31.1.1.1.15.119
13:56:04.613323 ens34 Out IP ubuntu-tiger-15-28.50537 > 212.183.148.10.snmp:  GetRequest(128)  system.sysName.0 interfaces.ifTable.ifEntry.ifDescr.123 31.1.1.1.18.123 31.1.1.1.15.123 interfaces.ifTable.ifEntry.ifDescr.119 31.1.1.1.18.119 31.1.1.1.15.119
```
Maybe I am missing something... but I think this is why I have:

akvorado_inlet_core_flows_errors_total{error="SNMP cache miss",exporter="212.183.148.10"} 77327

Any help would be appreciated. Thanks

**Reply by vincentbernat** [reply]
After restarting the orchestrator, you need to restart the inlet. The configuration is only fetched during startup.
